Loading View Controllers.playground/Helpers.remap

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
Are these files required for this project? They include local paths, and seem inappropriate for OSS